source/connect.txt

```diff
@@ -266,6 +266,16 @@ following actions:
 
 - Press ``Ctrl`` + ``C`` twice.
 
+.. _mongosh-work-alike-deployments:
+
+Work-alike Deployments
```

Comment:
[suggestion]

To me, "non-genuine" is a clearer, more meaningful term than "work-alike". I'm not sure how widely used "work-alike" is in technical communications, but it was a new term to me.

Could we maybe edit the heading to be "non-genuine", and replace the "work-alike" terminology used in this paragraph with "non-genuine"?

Reply:
Done 

It's a pretty common term, the main advantage here is that it's more neutral than 'non-genuine'.